scripts/us_bea/states_gdp/states_gdp.csv

```diff
@@ -0,0 +1,3061 @@
+,Quarter,GeoId,millions_of_chained_2012_dollars,Quantity_index,millions_of_current_dollars
```

Comment:
What's the first column represent? I noticed the it's not sequential. Do they represent the filtered regions?

Reply:
The first column represents the quarter of the year (Q1, Q2, Q3 or Q4) of the measurement. It is represented as being the last month in the observation period (so Q1 is 03), as per @pradh 's suggestion.